virtual_drive/core.py

```diff
@@ -1281,6 +1281,9 @@ def __init__(self, port: int, dictionary_path: Optional[str] = None) -> None:
         self.internal_generator = VirtualInternalGenerator(self)
 
         self.socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        if os.name != "nt":
```

Comment:
Check if OS is linux

Reply:
Done [db84905](https://github.com/ingeniamc/ingenialink-python/pull/391/commits/db849054c96971aa40e6b4216c347b3bca9990aa)